.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+labels: bug
+---
+
+**Describe your environment** Describe any aspect of your environment relevant to the problem, including your platform, build system, version numbers of installed dependencies, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on master.
```

Comment:
```suggestion
**Describe your environment** Describe any aspect of your environment relevant to the problem, including your platform, build system, version numbers of installed dependencies, etc. If you're reporting a problem with a specific version of a library in this repo, please check whether the problem has been fixed on main branch.
```

Reply:
Need to fix this for .NET too :) !

Reply:
.NET is correct since it hasn't yet migrated to main branch.